From 65b31124191b53e65d2cb2ab7349144f219543da Mon Sep 17 00:00:00 2001
From: Prabhu Sundararaj <prabhu.sundarara@nxp.com>
Date: Mon, 11 May 2020 23:26:19 -0500
Subject: [PATCH] MGS-5186 Per Specification EGL_NATIVE_PIXMAP_KHR requires
 EGL_NO_CONTEXT

https://www.khronos.org/registry/EGL/extensions/KHR/EGL_KHR_image_pixmap.txt

Per Specification EGL_NATIVE_PIXMAP_KHR requires EGL_NO_CONTEXT.
So passing a valid context will result in error.

Signed-off-by: Prabhu Sundararaj <prabhu.sundarara@nxp.com>
---
 glamor/glamor_egl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/glamor/glamor_egl.c b/glamor/glamor_egl.c
index d3c678d6b..5385e7c90 100644
--- a/glamor/glamor_egl.c
+++ b/glamor/glamor_egl.c
@@ -233,7 +233,7 @@ glamor_egl_create_textured_pixmap_from_gbm_bo(PixmapPtr pixmap,
     glamor_make_current(glamor_priv);
 
     image = eglCreateImageKHR(glamor_egl->display,
-                              glamor_egl->context,
+                              EGL_NO_CONTEXT /*glamor_egl->context*/,
                               EGL_NATIVE_PIXMAP_KHR, bo, NULL);
     if (image == EGL_NO_IMAGE_KHR) {
         glamor_set_pixmap_type(pixmap, GLAMOR_DRM_ONLY);
-- 
2.17.1

